package.json

```diff
@@ -16,7 +16,7 @@
     "prelint": "yarn --network-timeout 1000000",
     "lint": "eslint .",
     "pretest": "yarn lint",
-    "test": "ava --serial --verbose test/unit",
+    "test": "ava --serial --verbose test/unit/commands/tools/tool-test.js",
```

Comment:
Did you mean to change this back to run all unit tests?

Reply:
Ops. Vestige of local testing, will fix.